### ISSUE TITLE
NPE comparing qname attribute to missing attribute

### ISSUE THREAD

**Issue by bbray** [xmlunit/xmlunit#32]
Found a minor issue today using 2.0.0-SNAPSHOT, here is a unit test that produces an NPE.
```java
	@Test
	public void temp() {
		String str1="<doc xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\"><effectiveTime xsi:type=\"IVL_TS\"></effectiveTime></doc>";
		String str2="<doc xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\"><effectiveTime></effectiveTime></doc>";
		Diff d = DiffBuilder.compare(Input.fromString(str1)).withTest(Input.fromString(str2)).build();
	}
```

**Comment by bodewig** [comment]
Thanks, fixed with 7ed02fb - I'll close this issue once I've ported the test (and the fix) to XMLUnit.NET.

**Comment by bbray** [comment]
Cool thanks!
